api_server.py

```
@@ -74,6 +85,34 @@ def format_hours(minutes: Optional[int]) -> Optional[str]:
     return f"{minutes / 60.0:.2f}h"
 
 
+def format_availability_data(row: Any, now: datetime) -> Dict:
```

Ruff (0.15.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `row`

(ANN401)